testsuite/features/profiles/cloud_aws/Kiwi/POS_Image-JeOS6_41/config.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
```

Comment:
Do we really need to still adding support for 4.1?
I think POS_Image-JeOS6_41 and JeOS7_41 is not anymore need.

Reply:
This is autogenerated. Not really my area :-)

Reply:
Maybe @aaannz knows about it?

Reply:
This is part of the retail images copied from [templates repo](https://github.com/SUSE/manager-build-profiles/tree/master/OSImage).
You, as QE, are deciding what to have in test suite, so if 4.1 is no longer tested, or will not be tested in AWS, there is no reason to have this copy there.

Reply:
I don't see anything related to SUMA version in https://github.com/SUSE/manager-build-profiles/tree/master/OSImage, I mean there is no such folder suffixed with 41, 42, Head, Uyuni.
From where they come? I guess indeed, we can get rid of xxx_41 folder.

Reply:
templates repo does not differentiated between SUMA/Uyuni version. Templates work for all of them and what is build is provided by activation key.
However in test suite, because channels are not synced (at least they were not before), there is a fork of these templates with URL to external repositories included in the XML. Because repos URLs are added to XML, then you need different templates for 4.1, 4.2, etc.

Reply:
I think the 4.1 testsuite also used the git URL to uyuni/master. So all testsuites (4.1, 4.2, 4.3, Head) use the templates from uyuni/master.
I think we should not remove them as long as 4.1 is still "supported".

Reply:
ok, I will remove it from the cloud repo